demo/checkbox-basic-demos.html

```diff
@@ -45,6 +45,32 @@ <h3>Disabled state</h3>
       </template>
     </vaadin-demo-snippet>
 
+    <h3>Helper text</h3>
+    <p>Use the <code>helper-text</code> attribute or add content to the <code>helper</code> slot to set helper content.</p>
+    <vaadin-demo-snippet id="text-field-basic-demos-helper-text">
+      <template preserve-content>
+        <vaadin-checkbox-group label="Helper text" theme="vertical"
```

Comment:
It took me a moment to realize that the text that said "Helper text" wasn't the helper text in the demo, but the label XD Maybe the label should say something else instead.

Reply:
Changed to "Label" instead.